src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  background: $primary-bg-color;
+  color: $primary-text-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+
+    margin: 0;
+    padding-top: 40px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $secondary-text-color;
+
+    margin: 0;
+    padding: 4px 0 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    padding: 24px 0 16px;
+
+    &-title {
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary-text-color;
+    }
+
+    .card__price-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  .card__button {
+    font-weight: 700;
```

Comment:
I think u can create variables, , looks like it was already used more than one tim

Reply:
you can even create mixin or placeholder for common text styles